mountpoint-s3/tests/fuse_tests/cache_test.rs

```diff
@@ -252,6 +276,20 @@ fn cache_write_read_base<Cache>(
     );
 }
 
+async fn cache_read_empty<Cache>(cache: Cache, test_name: &str)
```

Comment:
nit: does not belong to `fuse_tests` since we're using the cache directly here (but probably having a separate file for non-fuse tests of the cache is a worse trade-off)

Reply:
I don't think we should do it now, but it makes sense to have a proper 'all caches' test (later)